libtransmission/utils.cc

```diff
@@ -193,6 +196,15 @@ bool tr_file_save(std::string_view filename, std::string_view contents, tr_error
     {
         return false;
     }
+#ifndef _WIN32
+    // set file mode per settings umask()
+    else
```

Comment:
Please delete the `else` line.

Reply:
done. https://github.com/transmission/transmission/pull/7195/commits/4f10c20eecf8ec6c3180696786c94889e22aa216